edx-platform/pearson-theme/lms/templates/courseware/course_about.html

```diff
@@ -0,0 +1,528 @@
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from courseware.courses import get_course_about_section
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from edxmako.shortcuts import marketing_link
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.djangolib.markup import HTML
+from openedx.core.lib.courses import course_image_url
```

Comment:
openedx are local application imports, so they need to be placed before six imports with a empty line between them. @amalbas  [PEP8 imports](https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
Sorry, openedx it was need to be placed after six imports. @amalbas